specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ContainerAppsLabelHistory.json

```diff
@@ -0,0 +1,202 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-10-02-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/labelHistory": {
+      "get": {
```

Comment:
Are there "put" and "patch" operations?

Reply:
label history only has GET, to put or patch you have to use the container apps traffic section